Makefile

```diff
@@ -6,7 +6,7 @@
 TOP = .
 include $(TOP)/build.mk
 
-SUB_DIRS = utils ercc ecc_enclave ecc tlcc_enclave tlcc examples integration # docs
+SUB_DIRS = docker utils ercc ecc_enclave ecc tlcc_enclave tlcc examples integration # docs
```

Comment:
I must admit i liked having the docker stuff in utils (given that we already have it and kept docker stuff previously there) rather than putting it in the top-level ....

Reply:
sure, we can keep it in utils